packages/atomic/src/components/atomic-layout/atomic-layout.tsx

```diff
@@ -0,0 +1,33 @@
+import {Component, Element, Prop} from '@stencil/core';
```

Comment:
I made a reusable component here too, so we could switch the layout dynamically using the Prop.
I am also open to make an atomic-search-layout, atomic-agent-panel-layout, etc. 🤷

Reply:
I like having the `layout` prop here. Being able to change it dynamically means I'll be able to test locally using a simple tampermonkey script.